frontend/src/pages/api/nodes.ts

```diff
@@ -1,12 +1,13 @@
+import { NextApiHandler } from "next";
 import { ExplorerApi } from "../../libraries/explorer-wamp";
 
-export default async function (req, res) {
+const handler: NextApiHandler = async (req, res) => {
   try {
     let ip_address =
       req.headers["x-forwarded-for"] ||
       req.connection.remoteAddress ||
       req.socket.remoteAddress ||
-      req.connection.socket.remoteAddress;
+      req.connection.remoteAddress;
```

Comment:
There is already `req.connection.remoteAddress` two lines above. I don’t recall where I got this `req.connection.socket.remoteAddress` from, but if it does not exist, just drop it

Reply:
Ok, I'll drop it